src/views/messages/messages.scss

```diff
@@ -24,6 +24,10 @@
     color: $type-gray;
 }
 
+select {
```

Comment:
We try to avoid using bare element selectors (like `p`, `a` or in this case, `select`), and instead try to target element class names. We could add more `select` elements to the page in the future, and wouldn't want this `width` rule to apply to all of them.

Since the `select` you're targeting already has `100%` as its width, you should be able to adjust its container's width instead. It looks like that container has the class `.messages-title-filter`.